draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2718,110 +2811,96 @@ uCDN-A                         dCDN-B                         dCDN-C
+      <section title="Filtered Representations" anchor="filtered-collection-representation">
       <t>
-         The Filtered Collections by state Link object represents a link to a filtered collection
-        per <xref target="trigger-state">trigger state</xref>.
-         It is encoded as a JSON object containing the following name/value pairs:
+      The filtered representation object describes a particular filtered representation of triggers. 
+      Currently it supports filtering by trigger state and trigger label. In the future, further filtering capabilities may be added.
+      It is encoded as a JSON object containing the following name/value pairs:
       </t>
-           <t><list style="empty" hangIndent="3">
-      <t><list style="hanging" hangIndent="3"><t hangText="Name: status">
-           <vspace blankLines="1"/> 
-      Description: Trigger state matching the filtered collection.
-           <vspace blankLines="1"/>
-           Value: A trigger State, as defined in <xref target="trigger-state"/>.
-           <vspace blankLines="1"/>
-      Mandatory: Yes.
-           </t>
-        
-           <t hangText="Name: collection">
-           <vspace blankLines="1"/>
-      Description: Filtered collection URL corresponding to the state.
-           <vspace blankLines="1"/>
-           Value: A URL represented as a JSON string.
-           <vspace blankLines="1"/>
-      Mandatory: Yes.
-           </t>
-        
-      </list></t>
-           </list></t> 
-
-      </section>
+   
 
-      <section title="Filtered Collections by Label Link" anchor="filtered-collection-state-link">
-      <t>
-         The Filtered Collections by Label Link object represents a link to a filtered collection
-              per trigger label.
-         It is encoded as a JSON object containing the following name/value pairs:
+      <t><list style="empty" hangIndent="3">
+      <t><list style="hanging" hangIndent="3">
+      <t hangText="Name: state">
+      <vspace blankLines="1"/> 
+      Description: Trigger state matching the filtered collection representation.
+      <vspace blankLines="1"/>
+      Value: A trigger state, as defined in <xref target="trigger-state"/>.
+      <vspace blankLines="1"/>
+      Mandatory: One of "state" or "label" should be present.
       </t>
-           <t><list style="empty" hangIndent="3">
-      <t><list style="hanging" hangIndent="3"><t hangText="Name: label">
-           <vspace blankLines="1"/> 
+
+      <t hangText="Name: label">
+      <vspace blankLines="1"/> 
       Description: Trigger Label matching the filtered collection.
-           <vspace blankLines="1"/>
-           Value: A Trigger Label, as defined in <xref target="ci-trigger-resource"/>.
-           <vspace blankLines="1"/>
-      Mandatory: Yes.
-           </t>
+      <vspace blankLines="1"/>
+      Value: A Trigger Label, as defined in <xref target="ci-trigger-resource"/>.
```

Comment:
The target is incorrect here.  It should likely be "trigger-labels"

Reply:
No, not really. 4.1.4 Trigger Labels describes the concept, but the trigger label JSON definition is in 4.1 Trigger Resource

Reply:
Ok, understood.  This one just looked a little weird because the State attribute above references 4.1.5 Trigger States, but this one linked to something more general.  But I understand the rationale, and I'm okay with it.